### PR TITLE
add aias00 card

### PR DESCRIPTION
**Pull request by Aias00**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -114,7 +114,30 @@ <h2>A project for first-time contributions</h2>
         <!-- DO NOT modify the TEMPLATE directly, make a copy and paste it below -->
         <!-- Keep one line of space above and below your card -->
         <!-- ========= Paste YOUR CARD directly BELOW this line ========= -->
-
+        <!-- ________ *TEMPLATE: MAKE A COPY* Contributor card START ________  -->
```
Comment (TenTraicion) [inline index.html:117]:
```suggestion

        <!-- ________ *aias* Contributor card START ________  -->
```

File: index.html [hunk 2]
```diff
@@ -114,7 +114,30 @@ <h2>A project for first-time contributions</h2>
...
+        <div class="card">
+          <p class="name">hongyu liu</p>
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/aias_0" target="_blank">aias</a>
```
Comment (TenTraicion) [inline index.html:122]:
```suggestion
            <a href="https://www.twitter.com/aias_0" target="_blank">@aias_0</a>
```

File: index.html [hunk 3]
```diff
@@ -114,7 +114,30 @@ <h2>A project for first-time contributions</h2>
...
+              <li>
+                <a href="#https://spring.io/" target="_blank" title="Second Resource">spring</a>
+              </li>
+              <li>
+                <a href="#https://brew.sh/" target="_blank" title="Third resource">homebrew</a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <!-- ________*END TEMPLATE* Contributor card END ________  -->
```
Comment (TenTraicion) [inline index.html:140]:
```suggestion
        <!-- ________*aias* Contributor card END ________  -->
        
```

File: index.html [hunk 4]
```diff
@@ -114,7 +114,30 @@ <h2>A project for first-time contributions</h2>
...
+          <p class="contact">
+            <i class="fab fa-twitter"></i>
+            <a href="https://www.twitter.com/aias_0" target="_blank">aias</a>
+          </p>
+          <p class="about">I am a Java Enginer from China, and I want to learn more web development skills</p>
+          <div class="resources">
+            <p>3 Useful Dev Resources</p>
+            <ul>
+              <li>
+                <a href="#https://vercel.com/dashboard" target="_blank" title="First Resource">vercel</a>
```
Comment (TenTraicion) [inline index.html:129]:
```suggestion
                <a href="https://vercel.com/dashboard" target="_blank" title="First Resource">vercel</a>
```

File: index.html [hunk 5]
```diff
@@ -114,7 +114,30 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="#https://spring.io/" target="_blank" title="Second Resource">spring</a>
```
Comment (TenTraicion) [inline index.html:132]:
```suggestion
                <a href="https://spring.io/" target="_blank" title="Second Resource">spring</a>
```

File: index.html [hunk 6]
```diff
@@ -114,7 +114,30 @@ <h2>A project for first-time contributions</h2>
...
+              </li>
+              <li>
+                <a href="#https://brew.sh/" target="_blank" title="Third resource">homebrew</a>
```
Comment (TenTraicion) [inline index.html:135]:
```suggestion
                <a href="https://brew.sh/" target="_blank" title="Third resource">homebrew</a>
```

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@Aias00 Kindly make the recommended changes and commit. Thanks

**Comment by Aias00** [comment]
hi, i have done them all, pls review it for me, thx

**Comment by CandidDeer** [comment]
Thank you for your contribution and making the requested changes. Well done. Your PR has been merged and your card will be live in a moment.

**Pull request merged**